internal/reader/home_chain.go

```diff
@@ -358,6 +358,7 @@ type OCR3Config struct {
 	F                     uint8                   `json:"F"`
 	OffchainConfigVersion uint64                  `json:"offchainConfigVersion"`
 	OfframpAddress        []byte                  `json:"offrampAddress"`
+	RmnHomeAddress        []byte                  `json:"rmnHomeAddress"`
```

Comment:
This will cause all tests to fail because this is not on-chain yet. I think we should remove this for now and maybe add later when its finally in there?

Reply:
Would it be just empty? We are not using it (under the rmn feature flag).

It seems like ccip integ test are passing here: https://github.com/smartcontractkit/chainlink/pull/14500